modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -232,7 +234,20 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
             'Pending',
             'Pending and New',
             array(''=>'All', 'P'=>'Pending', 'N'=>'New', 'PN'=>'Pending or New')
-        ); 
+        );
+
+        //query to get scan types that exist for the project
+        $types_q = $DB->pselect(
+                "SELECT ID, Scan_type FROM mri_scan_type mri
+                 JOIN files f ON (f.AcquisitionProtocolID=mri.ID)", array()
```

Comment:
This will generate a PHPCS error about multi-line function calls needing to be on separate lines when the module is added to Travis.

The array parameter should be on a separate line.